### DISCUSSION TITLE
Any way to install Bromite SystemWebView despite not met requirements?

### DISCUSSION THREAD

**Discussion by zit-veggie** [bromite/bromite#1527] [Q&A] [answered]
[Verifying allowed webview packages](https://github.com/bromite/bromite/wiki/Installing-SystemWebView#verifying-allowed-webview-packages) states, Bromite SystemWebView can only be installed without
- `E: signature block` for `packageName="com.android.webview"`
- preinstalled package `com.android.webview`

Latter package unfortunately *does* exist on my system:
```
user@host:~$ adb shell "pm list packages | grep webview"
package:com.android.webview
```
Is there any way to still install Bromite SystemWebView?

I am currently on LineageOS 17.1, would have the possibility to update 18.1 (if that matters).

**Comment by nityy** [comment, marked as answer]
Despite the preinstalled webview with that particular signature, the rest of the procedure worked just fine for me on LineageOS 17.1 Unofficial. Also, I replaced webview in `/system/product/app/webview` so you should be fine with ignoring the following line written later in the instructions

> * go to the webview directory in /system/app/webview (if you do not have such directory, this procedure will not work)

You could always try Nanodroid's installer if this doesn't work out

**Reply by zit-veggie** [reply]
Thanks for confirming it works on LOS!
I'll give it a try in next spare time and also try out Nanodroid and SystemWebView installation as Magisk Module.
